### DISCUSSION TITLE
EV Charger power usage not correct?

### DISCUSSION THREAD

**Discussion by TommySharpNZ** [emporia-vue-local/esphome#164] [Q&A]
I have our EV charger on "Circuit 4" and as it's an OpenEVSE charger I also have it integrated into Home Assistant.
My issue is that when I compare the power draw reported by OpenEVSE vs my Emporia running ESPHome there is quite a difference in the reported amount.

I don't suppose anyone could suggest some things for me to work on to resolve this?
At peak charge the OpenEVSE report 3.57kw which is close to the max 3.6kw that the Nissan Leaf can charge at... Emporia just reports 3.0kw...

![image](https://user-images.githubusercontent.com/15130284/236705291-fe4cfc4a-33ff-4531-83f3-fb625325ab6f.png)

**Comment by flaviut** [comment]
I would start with voltage calibration for all phases: https://github.com/emporia-vue-local/esphome/discussions/143

And then I would perform a current calibration: https://github.com/emporia-vue-local/esphome/discussions/131
